src/main/java/io/fabric8/maven/docker/access/UrlBuilder.java

```diff
@@ -187,7 +187,7 @@ private static String encode(String param) {
         }
     }
 
-    private String stripSlash(String url) {
+    public static String stripSlash(String url) {
```

Comment:
I wouldn't expose that as public, but just do the strip directly when querying the API